File: front-end/src/app/models/usuario.ts
export class Usuario {

  id: number ;
  nome: string ;
  data_nascimento: Date ;
  cpf: string ;
  email: string ;
  senha: string ;
  chassi: string ;
  endereco: string ;
  numero: string ;
  bairro: string ;
  cidade: string ;
  estado: string ;



  public constructor(data:any) {
    if (data) {
      this.id = Number(data.id);
      this.nome = String(data.nome);

    }else{
      this.id = 0;
      this.nome =  '';

    }
  }
}

File: back-end/help-car/model/Usuario.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace help_car.model
{
    [Table("tbusuario")]
    public class Usuario
    {
   
        [Column("id")]
        public int? id { get; set; }
        public string? nome { get; set; }
        public string? cpf { get; set; }
        public DateTime? data_nascimento { get; set; }
        public string? email { get; set; }
        public string? senha { get; set; }
        public string? chassi { get; set; }
        public string? endereco { get; set; }
        public string? numero { get; set; }
        public string? bairro { get; set; }
        public string? cidade { get; set; }
        public string? estado { get; set; }
    }
}
